### DISCUSSION TITLE
导航菜单栏icon问题

### DISCUSSION THREAD

**Discussion by tamako33** [LIlGG/halo-theme-sakura#484] [Q&A]
### 目前使用的 Sakura 版本？

2.0.0

### 目前使用的 halo 版本？

2.11.2

### 建议/问题
 
我自己的网站：https://blog.tamako33.cc
是这样子 我在设置菜单栏的时候 halo的默认提示是  _支持所有 iconify 图标。例如 https://icones.netlify.app/_
我去到网站选择好图标后 复制好对应的iconify 填进去之后，刷新页面发现显示空白
![ZSKB$B4$J05P)~0GM4YUL11](https://github.com/LIlGG/halo-theme-sakura/assets/122194530/2b7d8455-be90-48fe-943a-190f49eae69c)

后面我查看文档说全局支持svg格式 我复制svg的格式依旧空白
后面更换本地调用发现还是空白，有点懵不知道怎么解决了，希望作者您能够给予一点帮助谢谢

以下是调用的代码格式 （直接复制的不知道有没有错）
为了显示我删去了头部的< 尾部的>
iconify
span class="iconify" data-icon="dashicons:admin-home" data-inline="false" style="color: #7e84cd"></span

svg
svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 20 20"><path fill="#7e84cd" d="m16 8.5l1.53 1.53l-1.06 1.06L10 4.62l-6.47 6.47l-1.06-1.06L10 2.5l4 4v-2h2zm-6-2.46l6 5.99V18H4v-5.97zM12 17v-5H8v5z"/></svg

本地
/upload/home.svg

**Comment by LIlGG** [comment]
只需要填写 `{Icon Set}:{Icon}` 的格式即可。
例如你上述提供的 svg 代码，只需要填写 `dashicons:admin-home` 至输入框即可。
如下图所示：

<img width="778" alt="image" src="https://github.com/LIlGG/halo-theme-sakura/assets/31335418/7654bf77-8b8f-4be6-98c5-a564290c912f">

而 `{Icon Set}:{Icon}` 可以直接在 https://icones.netlify.app/ 中复制，例如：

<img width="638" alt="image" src="https://github.com/LIlGG/halo-theme-sakura/assets/31335418/afe589eb-bb34-41a5-90e1-47a0dcdf16b7">

但目前是不支持设置字体颜色的。
